### PR TITLE
update set up sublime section title

### PR DESCRIPTION
**Pull request by martijnhoekstra**
make setup sublime section coincide with the four-step title: Set up ENSIME Sublime

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Java and SBT should now be installed. The next step is to install Sublime Text (
 
 See [Troubleshooting](#troubleshooting) below to check whether Sublime Text can see `java` and `sbt`.
 
-### Set up Sublime Text
+### Set up ENSIME Sublime
```
Comment (flurdy) [inline README.md]: Not sure, maybe change to '''Set up Sublime Text with Ensime Sublime'''?

Comment (martijnhoekstra) [inline README.md]: Then maybe change the "four steps" above accordingly?

Comment (flurdy) [inline README.md]: Good point. :smile:

**Comment by rorygraves** [comment]
@martijnhoekstra Agreed with the comments - please update the PR to reflect your throughts - thanks for the contribution.

**Comment by martijnhoekstra** [comment]
will do tomorrow; I edited this right in the online text editor, but I'll want to amend, so I need access to something that is not a phone

**Comment by rorygraves** [comment]
@martijnhoekstra Ping

**Comment by rorygraves** [comment]
@martijnhoekstra  Can you also please click this https://cla-assistant.io/ensime/ensime-sublime?pullRequest=92 - I don't think its an overly onerous CLA.

**Comment by fommil** [comment]
docs should be migrated to ensime.org, would you be able to help us with that @martijnhoekstra ?

**Pull request closed without merging**